components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/notifier/MarketplaceAssistantApiPublisherNotifier.java

```diff
@@ -33,6 +33,8 @@
 import org.wso2.carbon.context.CarbonContext;
 import org.wso2.carbon.apimgt.impl.utils.APIUtil;
 
+import static org.wso2.carbon.apimgt.impl.utils.APIUtil.getAPIMVersion;
```

Comment:
Why do we need a static import here? Note that `org.wso2.carbon.apimgt.impl.utils.APIUtil` is already imported in this file.